pkg/new-kernel/Dockerfile

```diff
@@ -176,7 +180,7 @@ RUN ./autogen.sh && \
         --with-linux-obj=/linux \
         --with-config=kernel  \
         --enable-linux-builtin  \
-        --host="${EVE_TARGET_ARCH}"-linux-musl --build="${EVE_BUILD_ARCH}"-linux-musl && \
```

Comment:
@famleebob , did you figure out why EVE_BUILD_ARCH is not reflecting the right architecture here? In theory you shouldn't need to change this variable....